examples/asr/speech_to_text_eval.py

```diff
@@ -76,6 +76,11 @@ class EvaluationConfig(transcribe_speech.TranscriptionConfig):
     dataset_manifest: str = MISSING
     output_filename: Optional[str] = "evaluation_transcripts.json"
 
+    # decoder type: ctc or rnnt, can be used to switch between CTC and RNNT decoder for Joint RNNT/CTC models
+    decoder_type: Optional[str] = None
```

Comment:
These two are already there in the base `transcribe_speech.TranscriptionConfig` - no need to add them again.
@nithinraok for later removal.